content/docs/sidebar.json

```diff
@@ -87,12 +87,12 @@
       {
         "slug": "data",
         "label": "Working with data",
-        "source": "user-guide/data.md"
+        "source": "data.md"
       },
       {
         "slug": "remote-projects",
         "label": "Working with remote projects and objects",
-        "source": "user-guide/remote-projects.md"
+        "source": "remote-projects.md"
```

Comment:
May want to remove these changes to avoid merge conflicts vs. #113 but no big deal either way.